pkg/mounter/mounter-s3fs.go

```diff
@@ -323,3 +334,37 @@ func (s3fs *S3fsMounter) formulateMountOptions(bucket, target, passwdFile string
 	}
 	return
 }
+
+func cleanupOfs3fsPasswordFile(target string) {
```

Comment:
Change the definition
`cleanupOfs3fsPasswordFile(target string)` --> `removeS3FSCredFile(credDir, mountPath string)`